src/components/common/error-boundary.tsx

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import React from 'react';
+import React, { ErrorInfo } from 'react';
```

Comment:
hmm the white space is added here. Does the prettier not been ran?

Reply:
Looks like current configuration of prettier checked only js files, I'll update it and add all of the changes to this pr

Reply:
I added all of the ts and tsx files corrected by prettier to this pr, so the files number a little bit increased